project/Dependencies.scala

```diff
@@ -14,7 +14,8 @@ object Dependencies {
   lazy val sprayJsonV = "1.3.2"
   lazy val akkaV = "2.4.9"
   lazy val slickV = "3.1.1"
-  lazy val googleClientApiV = "1.20.0"
+  lazy val googleClientApiV = "1.22.0"
+  lazy val googleApiV = "1.20.0"
```

Comment:
This name isn't specific enough. Instead of `googleApi`, how about `googleGenomicsServicesApi'?